volume_project/extract/2022/rescu_sqls/rescu_centreline.sql

```diff
@@ -0,0 +1,161 @@
+-- Make a table that links RESCU detectors with the centreline from 2022-07-05
+CREATE TABLE scannon.rescu_cent_20220705 AS (
```

Comment:
```
select  * from scannon.rescu_cent_20220705  
where cl_geom is null
```
There are 19 rows with no geom but with a centreline_id!

Reply:
I think those are the ones you manually inserted, can you do a update set geom for those ones?

Reply:
Yes, the 19 missing geoms are the ones I updated manually (I wanted to be able to identify them quickly). This is an intermediary table that only includes RESCU detectors with good data in 2022, so I think this should be implemented via

General Improvement Idea 4: make a RESCU to centreline equivalency table (consider expanding this to include other VDS types)

Reply:
New issue: https://github.com/CityofToronto/bdit_data-sources/issues/923